Seneca-CDOT/telescope
# Bundle all client-side javascript in dashboard

## Issue This PR Addresses
Closes #2768 



## Type of Change




## Description



**Changelog**
- Update Dockerfile to install dev deps as well
- Compile sass with vite
- Convert material-dashboard to .cjs (commonjs)
- Bundle all client-side js
- Move vite, npm-run-all and sass to dev deps.

## Steps to test the PR



- Go to status at /src/api/status
- Run `pnpm dev`
- Website is live at port 1111

Everything should be the same as [staging](https://dev.api.telescope.cdot.systems/v1/status/).

## Checklist
